AsyncDisplayKit/ASEditableTextNode.h

```diff
@@ -93,9 +93,16 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readwrite) UIEdgeInsets textContainerInset;
 
 /**
- @abstract The returnKeyType of the keyboard. This value defaults to UIReturnKeyDefault.
+ @abstract <UITextInputTraits> properties.
  */
-@property (nonatomic, readwrite) UIReturnKeyType returnKeyType;
+@property(nonatomic, readwrite, assign) UITextAutocapitalizationType autocapitalizationType; // default is UITextAutocapitalizationTypeSentences
+@property(nonatomic, readwrite, assign) UITextAutocorrectionType autocorrectionType;         // default is UITextAutocorrectionTypeDefault
+@property(nonatomic, readwrite, assign) UITextSpellCheckingType spellCheckingType NS_AVAILABLE_IOS(5_0); // default is UITextSpellCheckingTypeDefault;
```

Comment:
Let's go ahead and remove the NS_AVAILABLE because we only support back to 7.  Then the comment will line up.  I know this is what Apple's header has though.